### ISSUE TITLE
LG WebOS möglich?

### ISSUE THREAD

**Issue by TheRealTremes** [thkl/Homematic-Virtual-Interface#104]
Hallo! Für den Philips TV gibt es ja bereits ein Plugin. Ist es auch für einen LG TV mit WebOS möglich? Zb das man diesen über eine 19 TFB bedienen kann?

Für NodeRed zb gibt es sowas:
https://github.com/hobbyquaker/node-red-contrib-lgtv

**Comment by nubbel7404** [comment]
Jau das wäre nice @thkl

**Comment by HookHM** [comment]
Ja, HVL würde meinen LG TV auch gut tun. Ist das machbar @thkl ?

**Comment by LevelOne2k** [comment]
@thkl wäre das umsetzbar? Würde mich auch darüber freuen

**Comment by thkl** [comment]
Bestimmt aber dauert

**Comment by Kinzal16** [comment]
Hallo @thkl kann das noch kommen? Würde mich sehr freuen meinen LG TV über HVL steuern zu können.
